docs/actions/introduction.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Actions
+---
+
+# Actions
+
+> For now, actions are only available for PostgreSQL and MySQL
+
+![Example action](./images/example-action.png)
+
+Actions are entities in Metabase that let you build custom forms and business logic.
+
+## What are actions?
+
+Actions let you write parameterized SQL that writes back to your database. Actions can be attached to buttons on dashboards to create custom workflows. You can even publicly share the parameterized forms they generate to collect data.
+
+Here are a few ideas for what you can do with actions:
+
+- Create a customer feedback form and embed it on your website.
+- Mark the customer you’re viewing in a dashboard as a VIP.
+- Let team members remove redundant data.
+
+## Enabling actions
+
+To enable actions for a database connection, admins should click on the gear icon in the upper right and navigate to **Admin settings** > **Databases**, then click on the database you want to create actions for. On the right side of the connection settings form, toggle the **Model actions** option.
+
+For actions to work, the database user account (the account you're using to connect to the database) must have write permissions.
+
+## Who can use actions
+
+- **To create or edit an action**, a person must be in a group with Native query editing privileges for the relevant database.
+-  **To run an action**, all you need is view access to the action's model or dashboard.
+
+## Types of actions
+
+There are two types of actions:
+
+- [Basic](./basic.md)
+- [Custom](./custom.md)
+
+## Actions on dashboards
```

Comment:
Could reframe this as 'running actions' and clarify that you can either a) run actions directly from the model detail page or b) wire them up to dashboards